tools/cli/go-whisk/whisk/wskerror.go

```diff
@@ -34,6 +34,7 @@ const NO_DISPLAY_PREFIX         bool = false
 const APPLICATION_ERR           bool = true
 const NO_APPLICATION_ERR        bool = false
 const TIMED_OUT                 bool = true
+const UNABLE_CREATE_ACTION_ERR  string = "Unable to create action '{{.name}}': {{.err}}"
```

Comment:
@mdeuser this was the spirit of the intended change/this pr. I think @lzbj initially interpreted the issue differently. So we should decide: is it worth making the change of passing the string around by pointer? If so, then yes decouple the prs. If not, drop that commit and making the intended change for reducing the number of times an error message is used. For the latter, I defer to you and @dubeejw and @lzbj for the best way to do that.